frontend/lib/resolution/Resolver.cpp

```diff
@@ -3713,6 +3746,19 @@ MatchingIdsWithName Resolver::lookupIdentifier(
                      /* receiverScopeHelper */ helper,
                      ident->name(), config, ident->id());
 
+  // For an inheriting compiler-generated initializer, allow performing lookup
+  // as though we can see anything the superclass can.
+  if (m.isEmpty() && this->superInitClassType) {
+    const Scope* superClassScope =
+        scopeForId(context, this->superInitClassType->id());
+    m = lookupNameInScopeWithWarnings(
+                     context, superClassScope,
+                     /* methodLookupHelper */ fHelper,
+                     /* receiverScopeHelper */ helper,
+                     ident->name(), config, ident->id());
+  }
+
+
```

Comment:
Generally, this surprises me. We already have a mechanism to check parent scopes -- that's the `methodLookupHelper`, I believe, which contains a list of scopes for each parent class. This means that the lookup should've found things that are in the receiver scopes:

https://github.com/chapel-lang/chapel/blob/7879364ef26973d8643fb4b84c83a7846901e99c/frontend/lib/resolution/scope-queries.cpp#L1824-L1827

Why do you need to separately perform a lookup in the `superClasScope`?

I can see `doLookupInReceiverScopes` respecting parents, so surely if the parent type's scope was in the method helper, you'd find even outer variables and things declared in imports at the definition point of the superclass.